po-refresh

```diff
@@ -159,16 +149,6 @@ def run(context, verbose=False, **kwargs):
                 current_linguas.sort()
             local_branch = add_and_commit_lang(name, locale[0], "Add")
 
-    # Here we have logic to only include files that actually
-    # changed translations, and reset all the remaining ones
-    files = output("git", "ls-files", "--modified", "po/")
-    for name in files.splitlines():
-        if name.endswith(".po"):
-            if changed(name):
-                execute("git", "add", "--", name)
```

Comment:
Don't we need to `git add *.po` then?

Reply:
No, we don't need to do that.`task.branch()` already `git add`s all files here: https://github.com/cockpit-project/bots/blob/main/task/__init__.py#L352 

po-refresh calls it with `pathspec="po/"`.

I did test it locally:
```
cp tmp/weblate-repo/*.po /var/home/martin/upstream/cockpit/po/
HEAD is now at 92f03a398 po: Update from Fedora Weblate
M	po/fr.po
M	po/pl.po
M	po/zh_CN.po
```
and the resulting commit had all three fr/pl/zh_CN files.